iceoryx_binding_c/include/iceoryx_binding_c/types.h

```diff
@@ -35,7 +35,7 @@ struct iox_ws_storage_t_
 {
     // the value of the array size is the result of the following formula:
     // sizeof(WaitSet) / 8
-    uint64_t do_not_touch_me[2965];
+    uint64_t do_not_touch_me[5902];
```

Comment:
Best oxymoron ever :rofl: Can we please rename them to `i_want_to_be_touched_softly`?